src/blockchain/log-processors/completesets.ts

```diff
@@ -11,24 +12,45 @@ export function processCompleteSetsPurchasedOrSoldLog(db: Knex, augur: Augur, lo
   const account = log.account;
   refreshPositionInMarket(db, augur, marketId, account, (err: Error|null) => {
     if (err) return callback(err);
-    const completeSetPurchasedData = formatBigNumberAsFixed({
-      marketId,
-      account,
-      blockNumber,
-      transactionHash: log.transactionHash,
-      logIndex: log.logIndex,
-      tradeGroupId: log.tradeGroupId,
-      numPurchased: log.numCompleteSets,
+    db.first("minPrice", "maxPrice", "numTicks").from("markets").where({ marketId }).asCallback((err: Error|null, marketsRow?: Partial<MarketsRow<BigNumber>>): void => {
+      if (err) return callback(err);
+      if (!marketsRow) return callback(new Error("market min price, max price, category, and/or num ticks not found"));
+      const minPrice = marketsRow.minPrice!;
+      const maxPrice = marketsRow.maxPrice!;
+      const numTicks = marketsRow.numTicks!;
+      const tickSize = numTicksToTickSize(numTicks, minPrice, maxPrice);
+      const numCompleteSets = augur.utils.convertOnChainAmountToDisplayAmount(new BigNumber(log.numCompleteSets, 10), tickSize).toFixed();
+      const query = db.select("timestamp").from("blocks")
+      .where("blockNumber", blockNumber);
+      query.asCallback((err: Error|null, blocksRows?: Array<BlocksRow>): void => {
+        if (err) return callback(err);
+        let timestamp = null;
+        if (blocksRows && blocksRows.length) {
+          timestamp = blocksRows[0].timestamp;
+        }
+        const completeSetPurchasedData: CompleteSetsRow<string> = {
+          marketId,
+          account,
+          blockNumber,
+          timestamp,
+          eventName: log.eventName,
+          transactionHash: log.transactionHash,
+          logIndex: log.logIndex,
+          tradeGroupId: log.tradeGroupId,
+          numCompleteSets,
+          numPurchasedOrSold: numCompleteSets,
+        };
+        augurEmitter.emit(log.eventName, Object.assign({}, completeSetPurchasedData));
```

Comment:
Object.assign isn't necessary here since you aren't merging, you can just emit `completeSetPurchasedData`